### ISSUE TITLE
Improve interaction with CQRS and Event Sourcing frameworks like Axon Framework etc

### ISSUE THREAD

**Issue by pipinet** [quarkusio/quarkus#1753]

**Comment by feaster83** [comment]
Yes, we need an extension to support integration with the Axon Framework.

**Comment by pepperbob** [comment]
@feaster83 this hit the Axonframework issues list - would you mind having a glance at https://github.com/AxonFramework/AxonFramework/issues/1318 - maybe you could provide further useful information.

**Comment by feaster83** [comment]
> @feaster83 this hit the Axonframework issues list - would you mind having a glance at [AxonFramework/AxonFramework#1318](https://github.com/AxonFramework/AxonFramework/issues/1318) - maybe you could provide further useful information.

I moved to current code to its own github repo with the idea to convert it to a separate extension for Quarkus (like the Quarkus Camel extensions, but that work is not done yet. https://github.com/feaster83/quarkus-axonframework-extension
